src/directives/internal/class.js

```diff
@@ -63,9 +63,11 @@ function setObjectClasses (el, obj) {
   var keys = Object.keys(obj)
   for (var i = 0, l = keys.length; i < l; i++) {
     var key = keys[i]
-    if (obj[key]) {
-      addClass(el, key)
-    }
+    if (!obj[key]) continue
+
+    key.split(' ').forEach(function (k) {
```

Comment:
Unconditional split + forEach can be slow, and class bindings can be a hot spot so performance is important.

Let's do an `key.indexOf(' ') > 1` check first, then use a normal for loop.